theano/sandbox/cuda/opt.py

```diff
@@ -370,7 +370,7 @@ def local_gpu_split(node):
         if (input.owner and isinstance(input.owner.op, HostFromGpu) or
             any(c != 'output' and isinstance(c.op, GpuFromHost) for c, idx
                 in outs_clients)):
-            new_op = GpuSplit(node.op.len_splits)
+            new_op = GpuSplit(node.op._props_dict()['len_splits'])
```

Comment:
@nouiz ,  I will update the PR and will paste it if i have any errors. And yeah, understood the goal of the issue.